en-US/win10/samples/speechtranslator.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: Blinky Sample
+permalink: /en-US/win10/samples/speechtranslator.htm
+lang: en-US
+---
+
+#Build the SpeechTranslator Project
+
+###Component Lists:
+
+- 2 Rpi2 boards
+- 2 Rpi2 Power Supplies 
+- 2 [Headsets](https://www.microsoft.com/hardware/en-us/p/lifechat-lx-3000/JUG-00013) 
+- 2 Mouses 
+- 2 Ethernet cables 
+- 2 SD cards 
+- 2 HDMI monitors 
+- 2 HDMI cable  
+- 1 Router 
+- 1 SD card reader
+
+
+### Setup your hardware
+- Follow the [instruction](http://ms-iot.github.io/content/en-US/win10/SetupRPI.htm) to flush your sd cards
+- Insert the sd card and USB headsets into devices; Also Connect HDMI monitors and Ethernet cable to the devices; You can connect both Ethernet cable to the router, then the router will be connected to the Internet
+- Once all connected, power the devices on
+- You should see the device name and device ip on the screen

```

Comment:
I would defer to the setup pages instead of recreating them here. (i.e. the whole Setup your hardware section should go away)